cmd/syndi/main.go

```diff
@@ -11,21 +11,28 @@ import (
 	_ "github.com/go-sql-driver/mysql"
 )
 
-var configFile = flag.String("c", "config.yaml", "configuration file to use")
-
 // TODO 1: generate in one goroutine and import in another...
 // TODO 2: have concurrent clients for faster import...
 func main() {
+	// save command-line arguments
+	args := config.RunArgs{}
+	flag.StringVar(&args.Database, "db", "bitstamp_dev", "Database name to use")
+	flag.StringVar(&args.Host, "host", "localhost", "Database host to connect to")
+	flag.StringVar(&args.Password, "p", "root", "Database user's password")
+	flag.StringVar(&args.Port, "P", "28000", "Database port number")
+	flag.BoolVar(&args.Safe, "safe", false, "Whether foreign key checks are mandated")
+	flag.StringVar(&args.User, "u", "root", "Database user")
 	flag.Parse()
+	args.Tables = flag.Args()
```

Comment:
Nice to have:
Move flag definition (and possibly parsing) to a function so we can just:
```
args = defineAndParseFlags()
```

Reply:
perhaps later...